src/test/java/org/broadinstitute/consent/http/service/InstitutionServiceTest.java

```diff
@@ -57,23 +53,31 @@ public void testCreateInstitutionSuccess() {
         when(institutionDAO.findInstitutionById(anyInt())).thenReturn(mockInstitution);
         initService();
         Institution institution = service.createInstitution(mockInstitution, 1);
-        assertNotNull(institution);
+        Assertions.assertNotNull(institution);
     }
 
-    @Test(expected = IllegalArgumentException.class)
+    @Test
     public void testCreateInstitutionBlankName() {
         Institution mockInstitution = initMockModel();
         mockInstitution.setName("");
         initService();
-        service.createInstitution(mockInstitution, 1);
+        try {
+            service.createInstitution(mockInstitution, 1);
+        } catch (Exception e) {
+            Assertions.assertTrue(e instanceof IllegalArgumentException);
+        }
     }
 
-    @Test(expected = IllegalArgumentException.class)
```

Comment:
It looks like in JUnit5 you can do:

```
@Test
void testException() {

  Exception thrown = Assertions.assertThrows(Exception.class, () -> {
       someFunctionThatThrowsException();
  });
}
```

Reply:
That is nicer than the try/catch approach 👍🏽